index.html

```diff
@@ -58,10 +63,15 @@ <h1 class="section-titlef">Facebook 360</h1>
                 <p class="section-description0">A daily selection of privately personalized reads; no accounts or
                     sign-ups
                     required.</p>
+                <p class="section-description1">Exploring the future of media in Facebook's first Virtual Reality app; a
+                    place to discover and enjoy 360 photos and videos on Gear VR.</p>
                 <ul class="tags">
                     <li class="li11">
                         <article class="text1">html</article>
                     </li>
+                    <li class="li222">
+                        <article class="text22"> Ruby on rails </article>
+                    </li>
```

Comment:
- [ ] As mentioned by the previous reviwer https://github.com/Kidd254/Lawrence-s-Portfolio/pull/4#discussion_r1043846980 , Please fix the gaps between technology elements currently it is like :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206868205-6521c7a7-1bf5-4253-a835-53e2cd1331ed.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206868224-e6ad5a38-bea0-4936-8214-3ba503316612.png)